streams/src/main/java/org/apache/kafka/streams/processor/internals/AssignedStreamsTasks.java

```diff
@@ -282,15 +274,19 @@ RuntimeException closeZombieTasks(final Set<TaskId> lostTasks, final List<TopicP
 
         for (final TaskId id : lostTasks) {
             if (suspended.containsKey(id)) {
+                log.debug("Closing the suspended stream task {}.", id);
                 firstException.compareAndSet(null, closeSuspended(true, suspended.get(id)));
             } else if (created.containsKey(id)) {
+                log.debug("Closing the created stream task {}.", id);
                 firstException.compareAndSet(null, closeNonRunning(true, created.get(id), lostTaskChangelogs));
             } else if (restoring.containsKey(id)) {
+                log.debug("Closing the restoring stream task {}.", id);
                 firstException.compareAndSet(null, closeRestoring(true, created.get(id), lostTaskChangelogs));
             } else if (running.containsKey(id)) {
+                log.debug("Closing the running stream task {}.", id);
                 firstException.compareAndSet(null, closeRunning(true, running.get(id), lostTaskChangelogs));
             } else {
-                // task may have already been closed as a zombie and removed from all task maps
+                log.warn("Skipping closing the lost stream task {} as it was already removed.", id);
```

Comment:
Just some logging, but I left it in this PR instead of my other logging PR as it applies to this scenario (the warning here would have been helpful)